recipe/bld.bat

```diff
@@ -109,6 +111,31 @@ if "%ARCH%" == "64" ( set "OPENCV_ARCH=64")
 robocopy %LIBRARY_PREFIX%\x%OPENCV_ARCH%\vc%VS_MAJOR%\ %LIBRARY_PREFIX%\ *.* /E
 if %ERRORLEVEL% GEQ 8 exit 1
 
+rem patch the cmake files accordingly
+"%PYTHON%" "%RECIPE_DIR%\patch_cmake_targets.py" ^
```

Comment:
Makes sense. Would something like in [vcpkg](https://github.com/Microsoft/vcpkg/blob/master/ports/opencv/opencv-installation-options.patch) be acceptable?